js/script.js

```diff
@@ -0,0 +1,18 @@
+//Set the background to the img src to keep alt text on images
+var imgs = document.getElementById("img-container").getElementsByTagName('img');
+document.body.style.backgroundImage = "url(" + imgs[0].src + ")";
+document.body.style.backgroundSize = "100%";
+const numPhotos = imgs.length - 1;
```

Comment:
Hey I see you made this variable but I don't see it being used anywhere

Reply:
My original intentwas to use it in the nextPhoto function, but forgot to. Thank you!